services/file-storage-and-transfer/index.yml

```diff
@@ -301,6 +447,17 @@ questions:
       - answer: Level 3
         category_classes:
           - 3
+  - question: Do you need a DOI for the data?
```

Comment:
but there are ways to get DOIs on data other ways - is this the best way to phrase this?